src/transformers/integrations.py

```diff
@@ -676,24 +697,56 @@ def setup(self, args, state, model, **kwargs):
+            Whether or not to disable code saving. Set `WANDB_DISABLE_CODE=True` to disable.
+        - **WANDB_ENTITY** (`str`, *optional*, defaults to `None`):
+            The entity associated with your run. If you have run wandb init in the directory of your training script,
+            it will create a directory named wandb and will save a default entity which can be checked into source control.
+            If you don't want to create that file or want to override the file you can use the environmental variable.
+        - **WANDB_JOB_TYPE** (`str`, *optional*, defaults to `None`):
+            Specify the job type, like `"training"` or `"evaluation"` to indicate different types of runs.
+        - **WANDB_LOG_MODEL** (`bool`, *optional*, defaults to `False`):
+            Whether or not to log model as artifact at the end of training. Use along with
+            [`~transformers.TrainingArguments.load_best_model_at_end`] to upload best model.
+        - **WANDB_MODE** (`str`, *optional*, defaults to `None`):
+            If you set this to `"offline"` wandb will save your run metadata locally and not sync to the server.
+            If you set this to `"disabled"` wandb will turn off completely.
+            Defaults to `"online"`.
+        - **WANDB_NAME** (`str`, *optional*, defaults to `None`):
+            The human-readable name of your run. If not set it will be randomly generated for you.
+        - **WANDB_NOTES** (`str`, *optional*, defaults to `Training`):
+            Longer notes about your run. Markdown is allowed and you can edit this later in the UI.
+        - **WANDB_PROJECT** (`str`, *optional*, defaults to `huggingface`):
+            The project associated with your run. This can also be set with wandb init,
+            but the environmental variable will override the value.
+        - **WANDB_RESUME** (`bool`, *optional*, defaults to `never`):
+            By default this is set to `never`. If set to `auto` wandb will automatically resume failed runs.
+            If set to `must` forces the run to exist on startup.
+            If you want to always generate your own unique ids, set this to `allow` and always set WANDB_RUN_ID.
+        - **WANDB_RUN_GROUP** (`str`, *optional*, defaults to `None`):
+            Specify the experiment name to automatically group runs together.
+            For example, you might be doing cross validation, or you might
+            have multiple jobs that train and evaluate a model against different test sets.
+        - **WANDB_RUN_ID** (`str`, *optional*, defaults to `None`):
+            Set this to a globally unique string (per project) corresponding to a single run of your script.
+            It must be no longer than 64 characters. All non-word characters will be converted to dashes.
+            This can be used to resume an existing run in cases of failure.
+        - **WANDB_TAGS** (`str`, *optional*, defaults to `None`):
```

Comment:
```suggestion
        - **WANDB_RUN_GROUP** (`str`, *optional*):
            Specify the experiment name to automatically group runs together.
            For example, you might be doing cross validation, or you might
            have multiple jobs that train and evaluate a model against different test sets.
        - **WANDB_RUN_ID** (`str`, *optional*):
            Set this to a globally unique string (per project) corresponding to a single run of your script.
            It must be no longer than 64 characters. All non-word characters will be converted to dashes.
            This can be used to resume an existing run in cases of failure.
        - **WANDB_TAGS** (`str`, *optional*):
```